File: Coding Ninjas - CodeStudio/Basics Of C++ Guided Path/total_salary.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
	int basic;
    char grade;
    cin >> basic >> grade;
    int totalSalary = basic + round((0.2 * basic) + (0.5 * basic) - (0.11 * basic));
    switch(grade) {
        case 'A':
            totalSalary += 1700;
            break;
        case 'B':
            totalSalary += 1500;
            break;
        default:
            totalSalary += 1300;
    }
    cout << totalSalary << endl;
    return 0;
}

File: Coding Ninjas - CodeStudio/Basics of Java Guided Path/total_salary.java
import java.util.Scanner;
class Solution {
	
	public static void main(String args[]) {
		
		Scanner sc = new Scanner(System.in);
        
        int basic = sc.nextInt();
        char grade = sc.next().charAt(0);
        
        int allow;
        
        if(grade == 'A')
        	allow = 1700;
        else if(grade == 'B')
        	allow = 1500;
        else
        	allow = 1300;
        
        float hra = (float)basic / 100 * 20;
        float da = (float)basic / 100 * 50;
        float pf = (float)basic / 100 * 11;
            
        float totalSalary = basic + hra + da + allow - pf;
        
        System.out.println(Math.round(totalSalary));
		
	}
}